libobs/cmake/os-windows.cmake

```diff
@@ -56,7 +56,7 @@ target_sources(
     util/windows/window-helpers.h
 )
 
-target_compile_options(libobs PRIVATE $<$<COMPILE_LANGUAGE:C,CXX>:/EHc->)
+target_compile_options(libobs PRIVATE $<$<COMPILE_LANGUAGE:C,CXX>:/EHc-> /wd4244 /wd4267)
```

Comment:
I'm not really a fan of disabling these warnings for all of libobs just because of a vendored dependency. Can we perhaps move it into `deps/`?

Reply:
We can probably try, but it will require a "bit" more work.

Reply:
To be clear, I'm not a fan of it either.

Reply:
An attempt was made:
https://github.com/RytoEX/obs-studio/tree/really-update-libnsgif-and-move

However, while libobs compiles, other projects that link to libobs that do `#include <graphics/image-file.h>` (obs-filters, obs-transitions, etc.) complain that they cannot find nsgif.h. I'm not sure what to do about that.